controller/workitem_blackbox_test.go

```diff
@@ -727,14 +737,18 @@ func newRelationBaseType(spaceID, wit uuid.UUID) *app.RelationBaseType {
 	}
 }
 
-func minimumRequiredCreatePayload() app.CreateWorkitemsPayload {
-	spaceSelfURL := rest.AbsoluteURL(&http.Request{Host: "api.service.domain.org"}, app.SpaceHref(space.SystemSpace.String()))
+func minimumRequiredCreatePayload(spaceIDOptional ...uuid.UUID) app.CreateWorkitemsPayload {
```

Comment:
or we could have a `NewCreatePayload` function that takes some optional functions as an (variadic) argument, and these functions would customize the payload. Eg:
````
func NewCreatePayload(customizers ...CustomizeCreatePayload) app.CreateWorkitemsPayload {
    p := ...
    for _, customizer := range customizers {
        customizer(p)
    }
 }
````
with
````
type CreateWorkitemsPayloadCustomizer func(*app.CreateWorkitemsPayload)
````
and then
````
func CustomizeSpace(spaceID uuid.UUID) CreateWorkitemsPayloadCustomizer {
    return func(p *app.CreateWorkitemsPayload) {
        spaceSelfURL := rest.AbsoluteURL(&http.Request{Host: "api.service.domain.org"}, app.SpaceHref(spaceID.String()))
        p.Data.Relationships= &app.WorkItemRelationships{Space: app.NewSpaceRelation(spaceID, spaceSelfURL)}
}
````
and finally
````
c := minimumRequiredCreatePayload(CustomizeSpace(spaceID))
````

in which case, we could then build other customizers if needed. WDYT ?

Reply:
Sounds like a good idea but let's do that in a separate PR.